tests/conftest.py

```diff
@@ -35,7 +35,9 @@ def app():
 
 @pytest.fixture
 def testapp(app):
-    return TestApp(app)
+    _app = TestApp(app)
+    _app.authorization = ('Basic', ('username', 'password'))
```

Comment:
The alternative is to add authentication for all the POST requests. This way seemed like less work. What's the concern here? I'm open to changing things, but I'd like to understand why before I do.